bots/twitch/phantombot/egg-phantom-bot.json

```diff
@@ -17,38 +17,29 @@
             "container": "ubuntu:16.04",
             "entrypoint": "bash"
         }
     },
     "variables": [
         {
             "name": "Version",
-            "description": "",
+            "description": "latest = Latest Stable\r\nmaster = latest Github",
             "env_variable": "RELEASE_VERSION",
             "default_value": "latest",
             "user_viewable": 1,
             "user_editable": 1,
             "rules": "max:9"
         },
-        {
-            "name": "Download Path",
-            "description": "Using the latest nightly build:\r\nhttps:\/\/raw.githubusercontent.com\/PhantomBot\/nightly-build\/master\/PhantomBot-nightly-lin.zip\r\n\r\nWhen you use the latest nightly build you need to change the docker image to:\r\nquay.io\/parkervcp\/pterodactyl-images:debian_openjdk-13",
-            "env_variable": "DOWNLOAD_PATH",
-            "default_value": "",
-            "user_viewable": 1,
-            "user_editable": 1,
-            "rules": "max:512"
-        },
         {
             "name": "Twitch Channel Name",
             "description": "Please enter the bot's Twitch username",
             "env_variable": "CHANNEL_NAME",
             "default_value": "",
             "user_viewable": 1,
             "user_editable": 1,
-            "rules": "required|string|max:26"
```

Comment:
Sorry, @parker. Missed the Comment on the commit :/

Reason for the remove was, that Guys ask me, how to do this, becasue they want to host some bots for friends. I run also in this Situation, because i don't want any Tokens.

For the Bot you need Username and Bot name and for both the Tokens.